.github/workflows/kind_integration.yml

```diff
@@ -65,6 +68,35 @@ jobs:
         name: image-archives
         path: /home/runner/archives
   # todo: Keep in sync with `release.yml`
+  cli_windows_static_integration_tests:
+    name: Integration tests (windows)
+    runs-on: windows-latest
+    needs: [docker_build]
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Try to load cached Go modules
+      # actions/cache@v1.1.2
+      uses: actions/cache@70655ec
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Download image archives (Forked repositories)
+      # actions/download-artifact@v1
+      uses: actions/download-artifact@18f0f59
+      with:
+        name: image-archives
+    - name: Install CLI
+      run: |
+        # Copy the CLI out of the $ARCHIVES/cli folder.
+        cp image-archives\cli\windows\linkerd $HOME\linkerd.exe
```

Comment:
The code is self-explanatory so no need of the comment (also for `go test ` below).